### DISCUSSION TITLE
How to use `include` config?

### DISCUSSION THREAD

**Discussion by wai-lin** [unocss/unocss#1947] [Q&A]
I've been playing around with `@unocss/nuxt`, `nuxt3` and `@nuxt/content`. Unocss is not parsing the `.md` files in `content` folder.

So I tried unocss with `include` config as follow and it's not working.

```ts
// unocss.config.ts
import { defineConfig } from 'unocss'
import { unoConfig } from './unocss-configs'

export default defineConfig({
	include: [
		'./content/**/*.md'
	],
	presets: [
		presetWind(),
	],
	transformers: [
		transformerVariantGroup()
	],
	shortcuts: unoConfig.shortcuts,
	safelist: [
		// ...'btn btn-indigo btn-green btn-outline-indigo btn-outline-green'.split(' '),
	]
})
```

My custom shortcuts are not generating unless I define it in safelist. Is there any way to config Uno to read all the `.md` files in `content` folder like in `tailwindcss` config?

```markdown
// content/index.md

# Hello

<button class="btn btn-indigo">Button</button>
```

**Comment by zyyv** [comment]
https://github.com/unocss/unocss#scanning, have you try to use `@unocss-include`?

**Comment by sibbng** [comment]
Patterns listed in `include` option will only be used during transform step. If these files are not processed by your bundler classes inside them can not be extracted.

This can be fixed with #1963. When it's released, just replace the `include` word with `watchExternal` and it will work. But transformers like `transformerVariantGroup` will not be applied to these files.

**Reply by jorgv** [reply]
@sibbng @antfu 

trying the latest feature extracontent, there is a type error:

```
export default defineConfig({
  extraContent: [
    './theme.js',
  ],
```
![image](https://user-images.githubusercontent.com/32174909/211396512-7a9d25c3-2415-4093-ab51-6b962db3d99b.png)

Is there a workaround to fix it?

thanks

**Reply by sibbng** [reply]
`extraContent: { filesystem: ['...'] }`
